google/auth/external_account.py

```diff
@@ -389,6 +391,48 @@ def with_token_uri(self, token_uri):
         kwargs.update(token_url=token_uri)
         return self.__class__(**kwargs)
 
+    def introspect_token(self, request):
+        """Introspection of token info.
+
+        If the credential object has a token_info_url provided, we can
+        introspect token info by requesting that endpoint.
+
+        Returns:
+            Mapping: The active token meta-information returned by the
+                introspection endpoint.
+
+        Raises:
+            google.auth.exceptions.InvalidResource:
+                If no token_info_url in config.
+            google.auth.exceptions.ClientCertError:
+                If no client auth given.
+            google.auth.exceptions.InvalidOperation:
+                If no token provided.
+            google.auth.exceptions.UserAccessTokenError:
+                If the credentials are invalid or expired.
+            google.auth.exceptions.TransportError:
+                If an error is encountered while calling the token
+                introspection endpoint.
+        """
+        # Token info introspection following RFC7662 standard:
```

Comment:
Maybe move this above?

Reply:
Delete. Don't need to mention here. Because the implementation is in the Client and in the Client's comment it's already mentioned.